packages/react-component-library/src/components/DatePickerE/partials/StyledFloatingBox.tsx

```diff
@@ -0,0 +1,23 @@
+import { selectors } from '@defencedigital/design-tokens'
+import styled from 'styled-components'
+
+import { FloatingBox } from '../../../primitives'
+import { StyledArrow } from '../../../primitives/FloatingBox/partials/StyledArrow'
+import { StyledContent } from '../../../primitives/FloatingBox/partials/StyledContent'
+
+const { color } = selectors
+
+export const StyledFloatingBox = styled(FloatingBox)`
```

Comment:
I've left these as overrides for now as it's not yet clear if these will be directly applicable anywhere else.